src/components/TodoItem.tsx

```diff
@@ -0,0 +1,66 @@
+import cn from 'classnames';
+import { useState } from 'react';
+import { useTodoContext } from '../context/TodoContext';
+import { deleteTodo } from '../api/todos';
+import { Todo } from '../types/Todo';
+import { Error } from '../types/Error';
+
+interface Props {
+  key: number;
+  todo: Todo;
+}
+
+export const TodoItem: React.FC<Props> = ({ key, todo }) => {
```

Comment:
Remove the 'key' prop from the TodoItem component, as it is a reserved property in React and is not necessary in this case.